docs/developer-tools/payouts-dapp-prototype.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Payouts DApp prototype
+sidebar_position: 3
+---
+
+## Discover, track, and withdraw role tokens
+
+The **[payouts DApp prototype](https://github.com/input-output-hk/marlowe-payouts)** is an example of a decentralized application designed to help users discover, track, and withdraw tokens from Marlowe smart contracts that use role tokens. It enables holders of role tokens in Marlowe smart contracts to withdraw the received funds, simplifying the process of tracking and withdrawing their payouts. 
+
+Since role tokens can be ada handles, ada domains, or custom tokens, users have a lot of flexibility with how to use the payouts DApp component. 
```

Comment:
Unless there is a policy against including links to products, I suggest we include them. In any case, they should be capitalized.
```suggestion
Since role tokens can be [Ada Handles](https://handle.tools/), [Ada Domains](https://www.adadomains.io/), or custom tokens, users have a lot of flexibility with how to use the payouts DApp component. 
```